src/config.rs

```diff
@@ -2202,6 +2202,7 @@ pub struct BackupConfig {
     pub auto_tune_refresh_interval: ReadableDuration,
     #[online_config(submodule)]
     pub hadoop: HadoopConfig,
+    pub enable_streaming: bool,
```

Comment:
move this config *before* `hadoop`.

why do we have this field anyway?

Reply:
we don't need any more, I'll remove it.